test/components/form-draft/publish.spec.js

```diff
@@ -109,7 +109,7 @@ describe('FormDraftPublish', () => {
       await modal.setProps({ state: true });
       modal.get('input').should.be.visible();
       // Explanatory text
-      modal.findAll('.modal-introduction p').length.should.equal(3);
+      modal.findAll('.modal-introduction p').length.should.equal(4);
```

Comment:
I'm looking at these tests and realizing the "Would you like to proceed?" question shows up on all the publish modals now. I think it's meant to just go with the dataset diff section.

Reply:
Relatedly, there should be some tests in this modal of the dataset diff showing up here.

Reply:
I think "Would you like to proceed?" should be display for all scenarios in publish modal because we are just describing impact, not asking any confirmation and showing "proceed" and "cancel" button

Reply:
No strong feelings on my end! I think it's reasonable to show it all the time (as long as it's below the input for the version string). The modal can definitely get long though, which is something to think about. We could also ask issa to take a look.